src/index.ts

```diff
@@ -1,3 +1 @@
-import Emoji from './Emoji';
-
-export default Emoji;
+export * from './Emoji';
```

Comment:
TypeScript tells me that: 'export *' does not re-export a default.

I would prefer not to change the exporting API, so can we change this to export the component as default and everything else as named?

Reply:
@SeanMcP Oups I was a bit fast for this PR sorry for that 😅 For sure we should keep the current exporting API. I also implemented the other suggestions 👍